dali/operators/image/crop/bbox_crop.cc

```diff
@@ -27,7 +27,19 @@ labels match the boxes that remain, after being discarded with respect to the mi
 Be advised, when `allow_no_crop` is `false` and `thresholds` does not contain `0` it is good to increase `num_attempts` as otherwise
 it may loop for a very long time.)code")
     .NumInput(2)
+    .InputDoc(0, "BBoxes", "TensorList of floats",
+              "Bounding boxes to encode in `[l,t,r,b]` or `[x,y,w,h]` format")
```

Comment:
this will change as soon as I merge my BBox PR. I am no longer using the `ltrb` terminology, and now you can have an arbitrary order of the coordinates

Reply:
I will wait for your PR, and replace those docs, it's basically placeholder taken from the docstring (as is in other places...).